sample/src/main/res/layout/activity_main.xml

```diff
@@ -1,29 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:numberpicker="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:paddingBottom="@dimen/activity_vertical_margin"
+    android:orientation="vertical"
     android:paddingLeft="@dimen/activity_horizontal_margin"
-    android:paddingRight="@dimen/activity_horizontal_margin"
     android:paddingTop="@dimen/activity_vertical_margin"
-    android:orientation="vertical"
-    xmlns:numberpicker="http://schemas.android.com/apk/res-auto"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingBottom="@dimen/activity_vertical_margin"
     tools:context="com.travijuu.numberpicker.sample.MainActivity">
 
     <com.travijuu.numberpicker.library.NumberPicker
         android:id="@+id/number_picker_default"
         android:layout_width="130dp"
-        android:layout_height="40dp"/>
+        android:layout_height="40dp" />
 
     <com.travijuu.numberpicker.library.NumberPicker
         android:id="@+id/number_picker_custom"
         android:layout_width="130dp"
         android:layout_height="40dp"
+        numberpicker:custom_layout="@layout/number_picker_custom_layout"
+        numberpicker:max="10"
         numberpicker:min="0"
+        numberpicker:unit="1"
+        numberpicker:value="-5" />
```

Comment:
Will it create an error? Because the `min` value is `0` but you set a negative number (below 0) as the value.